source/_components/switch.rpi_servo.markdown

```diff
@@ -0,0 +1,45 @@
+---
+layout: page
+title: "Raspberry PI Servo Switch"
+description: "Instructions on how to use a servo connected to a Raspberry PI into Home Assistant as a switch."
+date: 2017-10-23 21:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: raspberry-pi.png
+ha_category: Switch
+ha_release: 0.57
+ha_iot_class: "Local Push"
+---
+
+
+The `rpi_servo` switch platform allows you to control a servo through the GPIOs of your [Raspberry Pi](https://www.raspberrypi.org/).
+
+The servo will have a 'default_position' which it will turn from to the 'position_on' when toggled on, and the 'postion_off' when toggled off. It will remain in these positions for 'enabled_duration' before returning to the 'default_position'. Set the duration to zero to have the servo rest in the on and off state instead.
+
+To use your Raspberry Pi's GPIO in your installation, add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+switch:
+  - platform: rpi_servo
+    ports:
+      11: TV - Remote
+      12: Garage door
+```
+
+Configuration variables:
```

Comment:
Please use the configuration tags as specified in our documentation:

https://home-assistant.io/developers/documentation/create_page/#configuration